modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/CollectionItemContext.js

```diff
@@ -134,7 +134,7 @@ const useGetFieldValue = () => {
 		[context.collectionItem]
 	);
 
-	if (context.collectionFields !== null && context.collectionItem !== null) {
```

Comment:
When the configuration field value is resolved, `context.collectionFields` is not present, but `collectionItem` has a valid value. I am not sure if removing that condition will produce any error.